### ISSUE TITLE
docs: add guide to integrate with 3rd party UI libraries

### ISSUE THREAD

**Issue by devCrossNet** [devCrossNet/vue-starter#220]
- docs for integrating vuetify / bootstrap / element or just one of them
- how to handle libraries that reference browser features

**Comment by Heimathandel** [comment]
That'd be really helpful. Especially vuetify

**Comment by devCrossNet** [comment]
Would appreciate some help 🙂

**Comment by glederer** [comment]
It seems the project was started with a very opiniated vision on the usage of "storybook generated components". I guess adding/replacing with other libs like vuetify (I had in mind here vuesax :) but it is the same question ) is not that trivila as thios mean completely cleaning the "src/shared/components" folder of its content in the boilerplate, am I right?

**Comment by devCrossNet** [comment]
@glederer you shouldn't care about deleting stuff. If you do not reference any shared-components they will not be bundled. And if you need one of them for some reason you just reference it again.

One reason it is not built with UI libraries is mentioned in the introduction: https://vue-starter.herokuapp.com/docs/introduction.html#this-is-not-just-another-starter-project

**TL;DR:** I didn't find any UI lib that really fits our way of working.

I'll start writing the guide and reference the PR here, would be good if some of you could proofread it before I merge it.

**Comment by devCrossNet** [comment]
@Heimathandel @glederer see #225

**Comment by glederer** [comment]
Hello,

I tested this locally on a fresh git clone, I did all steps as described in this page  except the "clean up" part, assuming that we change the main app.vue componenent so I guess this should work.

I still get this error when trying to run locally :

> npm run dev

End of build says :

`ERROR in ./src/app/app/App/App.vue?vue&type=style&index=0&lang=css& (./node_modules/vue-loader/lib??vue-loader-options!./src/app/app/App/App.vue?vue&type=style&index=0&lang=css&) 233:0
[1] Module parse failed: Unexpected character '@' (233:0)
[1] You may need an appropriate loader to handle this file type.
[1] |
[1] |
[1] > @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons');
[1] | @import "../../../../node_modules/vuetify/dist/vuetify.css";
[1] |
[1]  @ ./src/app/app/App/App.vue?vue&type=style&index=0&lang=css& 1:0-132 1:148-151 1:153-282 1:153-282
[1]  @ ./src/app/app/App/App.vue
[1]  @ ./src/app/app.ts
[1]  @ ./src/client/index.ts
[1]  @ multi webpack-hot-middleware/client ./src/client/index
[1] Child html-webpack-plugin for "index.html":
[1]          Asset      Size  Chunks  Chunk Names
[1]     index.html  1.41 MiB       0
[1]     Entrypoint undefined = index.html
[1]     [./node_modules/html-webpack-plugin/lib/loader.js!./src/index.template.html] 1.79 KiB {0} [built]
[1]     [./node_modules/lodash/lodash.js] 527 KiB {0} [built]
[1]     [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {0} [built]
[1]     [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 519 bytes {0} [built]
[1] i ｢wdm｣: Failed to compile.
[1] debug: null`

Am I am missing something? or is there some typo in the webpack config adaptations?

**Comment by devCrossNet** [comment]
it seems that the css-loader doesn't work. can you provide some code?

**Comment by devCrossNet** [comment]
I forgot a snippet, sorry

webpack.base.config.js:
```
...
      {
        test:   /\.css$/,
        loader: ['vue-style-loader', 'css-loader'],
      },
...
```

**Comment by glederer** [comment]
many thanks @devCrossNet, it does work now.
